### DISCUSSION TITLE
emby播放115的视频，加载速度很慢

### DISCUSSION THREAD

**Discussion by liquidsnake9** [AlistGo/alist#6691] [Q&A]
alist搭建在本地，115的webdav是302方式
emby播放115的视频，转圈要转20-60秒，才会开始播放，播放过程很流畅，请教下这加载速度和什么有关？如何加快这个加载速度？

**Comment by djniche** [comment]
看你是什么方式添加到的emby,如果是strm,添加到媒体库是不会有ffprobe媒体信息的,首次播放会预先执行ffprobe读取媒体信息,读取完成后才会播放

**Reply by liquidsnake9** [reply]
> 看你是什么方式添加到的emby,如果是strm,添加到媒体库是不会有ffprobe媒体信息的,首次播放会预先执行ffprobe读取媒体信息,读取完成后才会播放

的确是strm，是不是这样就无解

**Reply by JJJJCG** [reply]
有解决办法了吗

**Reply by AspirantH** [reply]
https://zy.hn/2024/05/23/clouddrive2-alist-strm-302-emby-1/
搜到一篇貌似有用的文章，期待试验后分享成功解决方案
